tests/test_distance_from_best.py

```diff
@@ -47,33 +54,56 @@ class SpikyModelFactory(ModelFactory):
+            )
+            evaluation_end_time = factory.LazyAttribute(
+                lambda o: _sql_add_days(o.model_rel.train_end_time, 32)
+            )
 
         class Precision100Factory(ImmediateEvalFactory):
             metric = 'precision@'
             parameter = '100_abs'
 
+        class Precision100FactoryMonthOut(MonthOutEvalFactory):
+            metric = 'precision@'
+            parameter = '100_abs'
+
         class Recall100Factory(ImmediateEvalFactory):
             metric = 'recall@'
             parameter = '100_abs'
 
-        Precision100Factory(model_rel=models['stable_3y_ago'], value=0.6)
-        Precision100Factory(model_rel=models['stable_2y_ago'], value=0.57)
-        Precision100Factory(model_rel=models['stable_1y_ago'], value=0.59)
-        Precision100Factory(model_rel=models['bad_3y_ago'], value=0.4)
-        Precision100Factory(model_rel=models['bad_2y_ago'], value=0.39)
-        Precision100Factory(model_rel=models['bad_1y_ago'], value=0.43)
-        Precision100Factory(model_rel=models['spiky_3y_ago'], value=0.8)
-        Precision100Factory(model_rel=models['spiky_2y_ago'], value=0.4)
-        Precision100Factory(model_rel=models['spiky_1y_ago'], value=0.4)
-        Recall100Factory(model_rel=models['stable_3y_ago'], value=0.55)
-        Recall100Factory(model_rel=models['stable_2y_ago'], value=0.56)
-        Recall100Factory(model_rel=models['stable_1y_ago'], value=0.55)
-        Recall100Factory(model_rel=models['bad_3y_ago'], value=0.35)
-        Recall100Factory(model_rel=models['bad_2y_ago'], value=0.34)
-        Recall100Factory(model_rel=models['bad_1y_ago'], value=0.36)
-        Recall100Factory(model_rel=models['spiky_3y_ago'], value=0.35)
-        Recall100Factory(model_rel=models['spiky_2y_ago'], value=0.8)
-        Recall100Factory(model_rel=models['spiky_1y_ago'], value=0.36)
+        class Recall100FactoryMonthOut(MonthOutEvalFactory):
+            metric = 'recall@'
+            parameter = '100_abs'
+
+        for add_val, PrecFac, RecFall in [
+            (0, Precision100Factory, Recall100Factory),
+            (-0.15, Precision100FactoryMonthOut, Recall100FactoryMonthOut)
+            ]:
+            PrecFac(model_rel=models['stable_3y_ago'], value=0.6+add_val)
+            PrecFac(model_rel=models['stable_2y_ago'], value=0.57+add_val)
+            PrecFac(model_rel=models['stable_1y_ago'], value=0.59+add_val)
+            PrecFac(model_rel=models['bad_3y_ago'], value=0.4+add_val)
+            PrecFac(model_rel=models['bad_2y_ago'], value=0.39+add_val)
+            PrecFac(model_rel=models['bad_1y_ago'], value=0.43+add_val)
+            PrecFac(model_rel=models['spiky_3y_ago'], value=0.8+add_val)
+            PrecFac(model_rel=models['spiky_2y_ago'], value=0.4+add_val)
+            PrecFac(model_rel=models['spiky_1y_ago'], value=0.4+add_val)
+            RecFall(model_rel=models['stable_3y_ago'], value=0.55+add_val)
```

Comment:
What is `RecFall`? Is it a symmetric portmanteau of 'Recall', 'Factory', and 'Recall' again?

Reply:
Oh, hah... meant for that to be `RecFac` (i.e., "Recall Factory")! I'll change that to be a little more clear.